product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/ScenarioTestBase.java

```diff
@@ -180,4 +194,33 @@ public Boolean call() throws Exception {
         };
     }
 
+    protected String getStringFromDocument(Document doc) throws TransformerException {
```

Comment:
Shall we describe what this method does?

Reply:
This method was removed by [https://github.com/wso2/product-ei/pull/2853](https://github.com/wso2/product-ei/pull/2853)